docs/user-guide/ze-install-configuring-ze.md

```diff
@@ -0,0 +1,83 @@
+# Configuring Zowe Explorer
+
+:::info Required role: Developer
+:::
+
+Configure Zowe Explorer in the settings file so the extension performs according to your preferences.
+
+To access Zowe Explorer settings:
+
+1. Click the **Settings** icon at the bottom of the **Activity Bar**.
+2. Select the **Settings** option.
+3. Open the **Extension** option listed in the **Commonly Used** menu.
+4. Select **Zowe Explorer** to access its settings.
+5. Scroll the list to find the setting that needs modification.
+
+## Modifying creation settings for data sets, USS files, and jobs
+
+1. In Zowe Explorer settings, scroll to a data set, USS file, or job setting type.
+2. Click the setting's corresponding **Edit in settings.json** link.
+
+    This opens the `settings.json` file in an **Editor** tab. (The suggestions widget also opens if the functionality is enabled.)
+
+3. Edit the settings in the file as needed.
+4. Save the file to keep changes.
+
+    ![Configure Zowe settings](../images/ze/ZE-Configuration2.gif)
+
+## Modifying temporary file location settings
+
+Change the default folder location where temporary files are stored:
+
+   1. Navigate to Zowe Explorer settings.
+   2. Under the data set, USS, or jobs settings that you want to edit, click the **Edit in settings.json** link.
+   3. Modify the following definition in the file:
+
+      ```json
+      "zowe.files.temporaryDownloadsFolder": {
+          "folderPath": "/path/to/directory"
+        }
```

Comment:
I just want to verify that this JSON is accurate for this setting - with the latest V3 release, if I navigate to the setting and copy the setting as JSON, its displayed as follows:

`"zowe.files.temporaryDownloadsFolder.path": "<my-temp-download-path>"`

Notice that it refers to `.path` rather than `.folderPath` - I will double-check to confirm, but I don't believe we support the format shown above.

Reply:
I can confirm that if I try to use this format in v3, it is not recognized as a valid option:
![image](https://github.com/zowe/docs-site/assets/86500639/c96153f5-4b5a-4585-af49-bfd4f9af3757)

I would suggest changing this to:
```json
"zowe.files.temporaryDownloadsFolder.path": "/path/to/directory"
```